### PR TITLE
update OMERO.web deployment jobs

### PR DESCRIPTION
**Pull request by atarkowska**
as stated in https://trello.com/c/JnvMNKY0/148-add-virtualenv-to-ci-master-omero-dev-merge-daily-web-deployment

http://www.openmicroscopy.org/site/support/contributing-staging/ci-omero.html

### CODE REVIEW

File: contributing/ci-omero.txt [hunk 1]
```diff
@@ -147,6 +160,7 @@ under the :jenkinsview:`DEV` view tab of Jenkins.
         #. Triggers :term:`OMERO-DEV-merge-build` and
            :term:`OMERO-DEV-merge-integration`
         #. Triggers :term:`OMERO-DEV-merge-deploy`
+        #. Triggers :term:`WEV-DEV-merge-deploy`
```
Comment (sbesson) [inline contributing/ci-omero.txt]: typo `s/WEV/WEB`

File: contributing/ci-omero.txt [hunk 2]
```diff
@@ -347,3 +383,16 @@ Jenkins.
            :file:`components/tools/OmeroWeb/target/reports` from
            :term:`OMERO-DEV-breaking-integration`,
         #. Generates pytest report
+
+    :jenkinsjob:`WEB-DEV-breaking-integration`
```
Comment (sbesson) [inline contributing/ci-omero.txt]: This job is not set yet, is the plan to add one (to run breaking robot tests) or should we use WEB-DEV-breaking-deploy for now?

File: contributing/ci-omero.txt [hunk 3]
```diff
@@ -221,6 +239,20 @@ under the :jenkinsview:`DEV` view tab of Jenkins.
            :term:`OMERO-DEV-merge-integration`,
         #. Generates pytest report
 
+    :jenkinsjob:`WEB-DEV-merge-integration`
```
Comment (sbesson) [inline contributing/ci-omero.txt]: Currently called `WEB-DEV-integration-deploy`

File: contributing/ci-omero.txt (hunk 2 above)
Comment (atarkowska) [inline contributing/ci-omero.txt]: no sorry it should be just breaking

File: contributing/ci-omero.txt [hunk 4]
```diff
@@ -171,19 +185,23 @@ under the :jenkinsview:`DEV` view tab of Jenkins.
...
         #. Builds OMERO.server and starts it
-        #. Runs the OMERO.java, OMERO.py and OMERO.web integration tests
+        #. Runs the OMERO.java and OMERO.py integration tests
         #. Archives the results
         #. Triggers downstream collection jobs:
            :term:`OMERO-DEV-merge-integration-broken`,
            :term:`OMERO-DEV-merge-integration-java`,
-           :term:`OMERO-DEV-merge-integration-python`,
-           :term:`OMERO-DEV-merge-integration-web`
+           :term:`OMERO-DEV-merge-integration-python`
```
Comment (sbesson) [inline contributing/ci-omero.txt:210]: Have these collection jobs been combined? Or are we planning to have Web tests running only on Python 2.6?

Comment (atarkowska) [inline contributing/ci-omero.txt:210]: I removed them from list here, but not yet from CI

Comment (atarkowska) [inline contributing/ci-omero.txt:210]: I think it make sense to do it when https://github.com/openmicroscopy/openmicroscopy/pull/4783 is merged in

**Comment by hflynn** [comment]
Is this good to merge or waiting on something further?

**Comment by joshmoore** [comment]
Let me review this along with the related PRs.

**Comment by joshmoore** [comment]
With the new spiders, these will need updating. Something to discuss on Tuesday with the team before getting into ansible.

**Comment by joshmoore** [comment]
Server names need updating.

**Comment by joshmoore** [comment]
4783 is merged now. What needs sorting out in terms of the collection jobs?

**Comment by atarkowska** [comment]
just pushed, lets get that reviewed tomorrow.

File: contributing/ci-omero.txt [hunk 5]
```diff
@@ -67,46 +69,50 @@ OMERO jobs
...
+        * http://web-dev-integration.openmicroscopy.org
 
     -   * Breaking
         * :term:`OMERO-DEV-breaking-deploy`
         * trout.openmicroscopy.org
         * 34064
-        * https://trout.openmicroscopy.org/breaking
-
+        * :term:`WEB-DEV-breaking-deploy`
+        * http://web-dev-breaking.openmicroscopy.org
```
Comment (will-moore) [inline contributing/ci-omero.txt:123]: At http://web-dev-breaking.openmicroscopy.org/webclient/login/ the server list there does not include 'trout' as would be expected from all the other 3 logins above. Would be nice to see 'trout' in that list so we know where it's connecting to.

File: contributing/ci-omero.txt [hunk 6]
```diff
@@ -36,17 +36,19 @@ OMERO jobs
         * :term:`OMERO-DEV-merge-deploy`
         * :term:`OMERO-DEV-breaking-deploy`
 
+    -   * Deploys the merge OMERO web
+        * :term:`WEB-DEV-merge-deploy`
+        * :term:`WEB-DEV-breaking-deploy`
+
```
Comment (will-moore) [inline contributing/ci-omero.txt:46]: Do we also need to list WEB-DEV-latest-deploy and WEB-DEV-integration-deploy in this table?

**Comment by will-moore** [comment]
Couple of comments added above. But otherwise looking good & seems to make sense.

File: contributing/ci-omero.txt (hunk 5 above)
Comment (atarkowska) [reply contributing/ci-omero.txt:123]: fixed now in https://github.com/openmicroscopy/management_tools/pull/186

**Comment by atarkowska** [comment]
@will-moore comments adressed

**Comment by pwalczysko** [comment]
Reads good. The last fixes worked as expected. Ready to merge.

**Pull request merged**